### PR TITLE
11 handle files

### PR DESCRIPTION
**Pull request by MohammadAlhalaq**
Add file 
 1. Handelhtml.js and make handle for CSS and HTML and other.
 2. Make vavicon.
 3. Add function handelhtml in router

### CODE REVIEW

File: src/router.js [hunk 1]
```diff
@@ -1,20 +1,17 @@
...
-  // const extType = {
-  // html: 'text/html',
-  //   css: 'text/css',
-  //   jpg: 'image/jpg',
-  //   ico: 'image/x-icon',
-  //   png:'image/png',
-  //   js: 'application/javascript',
-  // }
+
+  const arrayPath = endpoint.split('/');
```
Comment (fadeomar) [inline src/router.js:9]: remove repeated lines

File: public/index.html [hunk 2]
```diff
@@ -43,8 +44,8 @@ <h1 class="head-title">Data Base For Names</h1>
 </div>
 </main>
     <!-- js files -->
-    <script src="./js/request.js"></script>
-    <script src="./js/dom.js"></script>
+    <script src="./public/js/request.js"></script>
```
Comment (mossa-Sammer) [inline public/index.html:47]: remove request.js

File: src/router.js (hunk 1 above)
Comment (mossa-Sammer) [inline src/router.js:9]: filePath, use descriptive name

File: src/handlers/htmllinks.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+  const extType = {
+    html: 'text/html',
+    css: 'text/css',
+    jpg: 'image/jpg',
+    ico: 'image/x-icon',
+    png: 'image/png',
+    js: 'application/javascript',
+  };
+
+  const pathF = path.join(__dirname, '..', '..', ...endpoint.split('/'));
```
Comment (mossa-Sammer) [inline src/handlers/htmllinks.js:17]: change it to filePath

Comment (fadeomar) [inline src/handlers/htmllinks.js:17]: use variable you already declared

**Review by fadeomar — APPROVED** [review]

File: public/index.html (hunk 2 above)
Comment (MohammadAlhalaq) [reply public/index.html:47]: that's removed

**Review by mossa-Sammer — APPROVED** [review]

File: src/handlers/htmllinks.js (hunk 3 above)
Comment (MohammadAlhalaq) [reply src/handlers/htmllinks.js:17]: done

Comment (MohammadAlhalaq) [reply src/handlers/htmllinks.js:17]: done

**Review by mohqarmout — APPROVED** [review]

File: src/handlers/htmllinks.js (hunk 3 above)
Comment (mohqarmout) [reply src/handlers/htmllinks.js:17]: hurry up

**Pull request merged**